axelrod/tournament_manager_factory.py

```diff
@@ -1,6 +1,10 @@
-from tournament_manager import *
+from __future__ import absolute_import
+from collections import OrderedDict
+
 import axelrod.strategies
 
+from .tournament_manager import *
```

Comment:
It's absolute versus relative imports in Python 3, going all the way back to  PEP 0328. It's necessary for the imports to work, although something like `from axelrod.tournament_manager import *` would probably also work. You just can't leave off the dot altogether any longer.